guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Importing_a_Red_Hat_Subscription_Manifest_into_Server_{context}"]
-= Importing a Red{nbsp}Hat Subscription Manifest into {ProjectServer}
+= Importing a Red{nbsp}Hat subscription manifest into {ProjectServer}
```

Comment:
I assume "Subscription Manifest" is capitalized but your call.

Reply:
This seems generic usage. Leaving lower case.